armi/bookkeeping/db/database.py

```diff
@@ -1463,6 +1463,73 @@ def _applyComponentNumberDensitiesMigration(comps, unpackedData):
             c.p["nuclides"] = nuclides
             c.p["numberDensities"] = numberDensities
 
+    @staticmethod
+    def getCycleNodeAtTime(dbPath, startTime, endTime, errorIfNotExactlyOne=False):
```

Comment:
from a user perspective we typically are just looking for 1 time node, so i would prefer if this defaulted to "true"